src/main/java/org/folio/dew/batch/bulkedit/jobs/BulkEditItemProcessor.java

```diff
@@ -112,16 +111,19 @@ public ItemFormat process(Item item) {
       .permanentLocation(isEmpty(item.getPermanentLocation()) ? EMPTY : item.getPermanentLocation().getName())
       .temporaryLocation(isEmpty(item.getTemporaryLocation()) ? EMPTY : item.getTemporaryLocation().getName())
       .effectiveLocation(isEmpty(item.getEffectiveLocation()) ? EMPTY : item.getEffectiveLocation().getName())
-      .inTransitDestinationServicePoint(itemReferenceService.getServicePointNameById(item.getInTransitDestinationServicePointId(), errorServiceArgs))
       .statisticalCodes(fetchStatisticalCodes(item, errorServiceArgs))
-      .purchaseOrderLineIdentifier(item.getPurchaseOrderLineIdentifier())
       .tags(isEmpty(item.getTags()) ? EMPTY : String.join(ARRAY_DELIMITER, escaper.escape(item.getTags().getTagList())))
-      .lastCheckIn(lastCheckInToString(item, errorServiceArgs))
       .build();
     itemFormat.setElectronicAccess(electronicAccessService.getElectronicAccessesToString(item.getElectronicAccess(), errorServiceArgs));
     return itemFormat.withOriginal(item);
   }
 
+  private String getInstanceTittle(Item item) {
```

Comment:
@AntonAntonich please fix misprint for 'tittle '

Reply:
fixed